src/components/SideBar/SideBar.tsx

```diff
@@ -23,11 +23,19 @@ import SkeletonSideBarList from './SkeletonSideBarList'
 import ToggleSideBar from './ToggleSideBar'
 import './sideBar.css'
 
+export interface ListChildrenSubItemsProps {
+  childrenSubItem: {
+    title: string
+    goTo: () => void
+    active: boolean
+  }[]
+}
 export interface SideBarSubItem {
   [key: string]: {
     title: string
     active: boolean
     goTo: () => void
+    childrenSubItem?: ListChildrenSubItemsProps
   }
 }
```

Comment:
i think we can reuse some cases:

```ts
export type SubItem = {
 title: string
 active: boolean
 bgActive?: string
 colorActive?: string 
 goTo: () => void
}

export interface SideBarSubItem {
  [key: string]: SubItem & { subItems: { [key: string]:  SubItem } }
}
```

and now, we can access to sub-sub items and can reusing types